presto-docs/src/main/sphinx/release/release-0.122.rst

```diff
@@ -8,3 +8,10 @@ General Changes
 * The deprecated casts between JSON and VARCHAR will now fail and provide the
   user with instructions to migrate their query. For more details, see
   :doc:`/release/release-0.116`.
+* Fixed bug in ``GROUP BY`` for structural types
```

Comment:
This needs more explanation. Did it not work for certain things? Incorrect behavior? Bad performance?